syntax = "proto3";
package nexus.hub.proto;

import "idm/api/proto/chat_manager_service.proto";

message SubmitRequest {
    bytes telegram_document = 1;
    string telegram_file_id = 2;
    idm.api.proto.Chat chat = 3;
    string bot_name = 4;
}
message SubmitResponse { }

service Submitter {
    rpc submit(SubmitRequest) returns (SubmitResponse) {};
}
